alarm_control_panel-card/alarm_control_panel-card.js

```diff
@@ -267,6 +268,14 @@ class AlarmControlPanelCard extends HTMLElement {
   _generateButton(state) {
     return `<paper-button raised id="${state}">${this._translateState(state)}</paper-button>`;
   }
+  
+  _generateDigitButton(digit, letters, displayText) {
+    if (displayText && letters != null) {
+      return `<paper-button data-digit="${digit}" raised style="text-align: center;">${digit}<br/>${letters}</paper-button>`
```

Comment:
Duplication should be avoided for paper-button, better to construct a string based on smaller pieces. 
Using <br/> for managing layout is to be avoided. Use CSS to align elements (especially when getting to <br/><br/>. This will break worse when we'll try to resize the card for `panel: true` with proper scaling.
Try adding letters in their own container, so they can be styled separately.

Reply:
CSS...eck. This one might be above my pay grade. I'll try to poke around some more. Thanks.

Reply:
I will assist next week once #electriccastle is over

Reply:
I think it's best to redo the PR if we get refactor from @gwww merged in.